### ISSUE TITLE
Bulk deletion of licenses

### ISSUE THREAD

**Issue by ssydorov** [snipe/snipe-it#7486]
**Server (please complete the following information):**
 - Snipe-IT Version : 4.7.7
 - OS: [ CentOS]
 - Web Server: [Apache]
 - PHP Version : 7

Is it possible to make a bulk deletion of licenses? I did a mistake during import. Thanks.

**Comment by ssydorov** [comment]
yes, it's still actual. But for future needs. At this moment I deleted all licenses manually.

**Comment by SupersecretIT** [comment]
Would also like to see this implemented, licenses especially would likely be a hassle to delete since they need to be for each individual for cost calculations to work.

**Comment by hrconsultJP** [comment]
The hours I just spent removing licenses, one by one, says this is still relevant.  Would be very much appreciated so I do not have to do that again.

**Comment by snipe** [comment]
Does the command line tool not work for this? https://snipe-it.readme.io/docs/bulk-license-checkoutcheckin

<blockquote><img src="/favicon.ico" width="48" align="right"><div>Snipe-IT Documentation</div><div><strong><a href="https://snipe-it.readme.io/docs/bulk-license-checkoutcheckin">Bulk License Checkout/Checkin</a></strong></div><div>If you need to check out a particular license to all users (for example an Office 365 license, which every user in your organization should theoretically have) or bulk checkin a license from all users (for example if you stopped using G-Suite), there is a console command to let you do that:andThe --...</div></blockquote>

**Comment by TRS-GitHub** [comment]
Can we get an answer to this?

**Comment by scottmeikle** [comment]
We have a lot of retail licences (>300) for old Microsoft Office products that we no longer use. I'd like to remove these old licences and a bulk delete feature would be a big help.

**Comment by snipe** [comment]
This was added a while back
